src/workspaces/common/state/useWorkspace.ts

```diff
@@ -22,7 +22,7 @@ export type { InitializedWorkspaceWrapper } from 'src/libs/state';
 export interface StorageDetails {
   googleBucketLocation: string; // historically returns defaultLocation if bucket location cannot be retrieved or Azure
   googleBucketType: string; // historically returns locationTypes.default if bucket type cannot be retrieved or Azure
-  fetchedGoogleBucketLocation: 'SUCCESS' | 'ERROR' | undefined; // undefined: still fetching
+  fetchedGoogleBucketLocation: 'SUCCESS' | 'ERROR' | 'RPERROR' | undefined; // undefined: still fetching
```

Comment:
Changing the model to distinguish between requester-pays errors with `RPERROR` and other errors with `ERROR`